tests/test_evpn_tunnel.py

```diff
@@ -0,0 +1,749 @@
+        vxlan_obj.check_vlan_extension(dvs, '100', '7.7.7.7')
+        print("Pass")
+
+        print("Testing Vlan Extension removal")
+        remove_evpn_remote_vni(dvs, 'Vlan100', '7.7.7.7')
+        remove_evpn_remote_vni(dvs, 'Vlan101', '7.7.7.7')
+        vxlan_obj.check_vlan_extension_delete(dvs, '100', '7.7.7.7')
+        vxlan_obj.check_vlan_extension_delete(dvs, '101', '7.7.7.7')
+        print("Testing DIP tunnel not deleted")
+        vxlan_obj.check_vxlan_dip_tunnel(dvs, tunnel_name, '6.6.6.6', '7.7.7.7')
+        print("Pass")
+
+        print("Testing Last Vlan removal and DIP tunnel delete")
+        remove_evpn_remote_vni(dvs, 'Vlan102', '7.7.7.7')
+        vxlan_obj.check_vlan_extension_delete(dvs, '102', '7.7.7.7')
+        vxlan_obj.check_vxlan_dip_tunnel_delete(dvs, '7.7.7.7')
+        print("Pass")
+
+        print("Testing Last Vlan removal and DIP tunnel delete for 8.8.8.8")
+        remove_evpn_remote_vni(dvs, 'Vlan100', '8.8.8.8')
+        vxlan_obj.check_vlan_extension_delete(dvs, '100', '8.8.8.8')
+        vxlan_obj.check_vxlan_dip_tunnel_delete(dvs, '8.8.8.8')
+        print("Pass")
+
+        remove_vxlan_tunnel_map(dvs, tunnel_name, map_name, '1000', 'Vlan100')
+        remove_vxlan_tunnel_map(dvs, tunnel_name, map_name_1, '1001', 'Vlan101')
+        remove_vxlan_tunnel_map(dvs, tunnel_name, map_name_2, '1002', 'Vlan102')
+        vxlan_obj.check_vxlan_tunnel_map_entry_delete(dvs, tunnel_name, vlanlist, vnilist)
+
+        print("Testing SIP Tunnel Deletion")
+        remove_evpn_nvo(dvs, 'nvo1')
+        remove_vxlan_tunnel(dvs, tunnel_name)
+        vxlan_obj.check_vxlan_sip_tunnel_delete(dvs, tunnel_name)
+
+
+#    Test 3 - Create and Delete SIP Tunnel and Map entries
+#    @pytest.mark.skip(reason="Starting Vxlanmgr to be merged")
+    def test_p2mp_tunnel_with_dip(self, dvs, testlog):
+        vxlan_obj = self.get_vxlan_obj()
+
+        tunnel_name = 'tunnel_2'
+        map_name = 'map_1000_100'
+        map_name_1 = 'map_1001_101'
+        map_name_2 = 'map_1002_102'
+
+        vxlan_obj.fetch_exist_entries(dvs)
+
+        create_vlan1(dvs,"Vlan100")
+        create_vlan1(dvs,"Vlan101")
+        create_vlan1(dvs,"Vlan102")
+        create_vxlan_tunnel(dvs, tunnel_name, '6.6.6.6', '2.2.2.2', False)
+        create_vxlan_tunnel_map(dvs, tunnel_name, map_name, '1000', 'Vlan100')
+        create_vxlan_tunnel_map(dvs, tunnel_name, map_name_1, '1001', 'Vlan101')
+        create_vxlan_tunnel_map(dvs, tunnel_name, map_name_2, '1002', 'Vlan102')
+
+        vlanlist = ['100', '101', '102']
+        vnilist = ['1000', '1001', '1002']
+
+        print("Testing SIP Tunnel Creation")
+        vxlan_obj.check_vxlan_sip_tunnel(dvs, tunnel_name, '6.6.6.6', vlanlist, vnilist, '2.2.2.2', False)
+        print("Pass")
+
+        print("Testing Tunnel Map Entry")
+        vxlan_obj.check_vxlan_tunnel_map_entry(dvs, tunnel_name, vlanlist, vnilist)
+        print("Pass")
```

Comment:
Suggest removing the print("Pass") statements

Reply:
Done